### ISSUE TITLE
After starting, mockoon is now default application for *.html files.

### ISSUE THREAD

**Issue by vladimir-light** [mockoon/mockoon#977]
**Describe the bug**

All HTML files opens with mockoon desktop app. Yes, not with your default browser like Chrome/FF/similar, but with mockoon. Mockoon is simply launched as if you launched the app from the launcher.

After a long search I found out that mockoon (for whatever reason) changes the default settings in config of my OS after startup.

##### Before mockoon start

```sh
cat ~/.local/share/applications/mimeapps.list
```

```
[Added Associations]
text/html=google-chrome.desktop
x-scheme-handler/mailto=thunderbird.desktop;
message/rfc822=thunderbird.desktop;
application/x-extension-eml=thunderbird.desktop;

[Default Applications]
text/html=google-chrome.desktop
```

```sh
cat ~/.config/mimeapps.list
```

```
[Default Applications]
x-scheme-handler/webcal=google-chrome.desktop
x-scheme-handler/postman=Postman.desktop
x-scheme-handler/mockoon=mockoon.desktop
```

##### After mockoon start

`~/.config/mimeapps.list` changes to

```
[Default Applications]
x-scheme-handler/webcal=google-chrome.desktop
x-scheme-handler/postman=Postman.desktop
x-scheme-handler/mockoon=mockoon.desktop
text/html=mockoon.desktop <--- this was added
```

Click on any HTML file → mockoon app launches.
This behavior "resets to normal" after each reboot.

**To Reproduce**

1. Install from *.deb package
2. Click on any HTML file and see your default browser 
2.1 `cat ~/.config/mimeapps.list`. 
3. Run the Mockoon app and close it (or leave it open)
4. Click on any HTML file and see mockoon start up
4.1 `cat ~/.config/mimeapps.list` again

**Mockoon version:**

noticed while using Desktop-App  `1.22` and `1.23`

**OS / OS version:**

OS: Ubuntu 20.04.5 LTS x86_64
Kernel: 5.15.0-67-generic 
Terminal: gnome-terminal 
CPU: Intel i7-10750H (12) @ 5.000GHz 
GPU: NVIDIA Quadro T1000 Mobile 
GPU: Intel UHD Graphics 
Memory: 13718MiB / 31893MiB 

Any Ideas?

**Comment by 255kb** [comment]
Thank you for reporting. 
We are not directly requesting any file association, I suspect this is due to either Electron on the electron-builder we are using for packaging. I found old similar issues. 
I will investigate.

**Comment by iXingo** [comment]
Same here on ubuntu 22.04

**Comment by 255kb** [comment]
Not very lucky yet, but it seems to be a [bug from xdg-settings](https://gitlab.freedesktop.org/xdg/xdg-utils/-/issues/180) that Electron is using to set the custom URL schemes. 
Some mentions [here](https://github.com/electron/electron/issues/20382), [here](https://github.com/aunetx/deezer-linux/issues/13) and [here](https://github.com/valerie-makes/notion-linux/issues/42) but it doesn't seem to be fixed. One person is suggesting a [workaround](https://github.com/valerie-makes/notion-linux/issues/42#issuecomment-818722236).

While waiting for a fix, I will disable the custom protocol on Linux. It's not that critical.

**Comment by 255kb** [comment]
This has been released in [v3.0.0](https://mockoon.com/releases/3.0.0/).
